### ISSUE TITLE
[0.5][Style-Guide] Update components inheritance

### ISSUE THREAD

**Issue by gibbsfromncis** [bitshares/bitshares-ui#2197]
The issue is that I used the following way now (I did wrap of components instead of inheritance):

```
class Input extends React.Component {

    static propTypes = {
        unsaved: PropTypes.bool,
    };

    render() {

        const props = {...this.props};

        return (
            <AntInput {...props}/>
        );
    }

}
```

And this way blocks us from using `properties` and `methods` of Ant Input. The simple example is:

```
componentDidUpdate(){
  this.refs.username.focus();
}

render(){
   return (<Input ref="username" />);
}
```

The focus is not a method of `Input` component, because it is method of `Ant Input`. 

The idea of this issue is to update all style-guide components to use inheritance instead of wrapping.

**Comment by gibbsfromncis** [comment]
@sschiessl-bcp I decided to use simple "forwarded" of ant components. The reason why I decided to use inheritance or wrapping if on some point we want to provide additional props to ant components. We already use Style-Guide for more than 4 months and we didn't have any requirements to this kind of feature. So to take it works fast and do not put much effort I just "forwarded" ant components.

Commit: https://github.com/bitshares/bitshares-ui-style-guide/commit/d3de9dfcdb2a0d3a3238326b2a0b46ec700250f0
